src/weather-bar.ts

```diff
@@ -237,6 +273,11 @@ export class WeatherBar extends LitElement {
       max-width: max(0px, calc((100% - 40px) * 999));
       overflow: hidden;
     }
+    .condition-icon-big {
```

Comment:
I don't like this class, it introduces some unnecessary duplication. But not sure if there is a better way to override `max-width` from `.condition-icon`.